libs/shared/contentful/src/lib/factories.ts

```diff
@@ -71,3 +71,9 @@ export const ContentfulCDNErrorFactory = (
   requestId: faker.string.uuid(),
   ...override,
 });
+
+export const MetadataFactory = (
+  override?: Partial<StripeMetadata>
+): StripeMetadata => ({
+  ...override,
+});
```

Comment:
Should this be moved to the Stripe lib?

Reply:
I had it in there initially, but since the StripeMetadata type is specific to the Contentful mapping function, it felt like it should be separate from the stripe library.